app/src/main/AndroidManifest.xml

```diff
@@ -12,11 +12,10 @@
 		    android:label="@string/app_name"
 		    android:roundIcon="@mipmap/ic_launcher_round"
 		    android:supportsRtl="true"
-		    android:theme="@style/AppTheme"
+		    android:theme="@style/RegistrationTheme"
```

Comment:
Jak chcesz zmienić theme fragmentu, to zmień theme fragmentu, nie całej aplikacji